src/pybind/mgr/dashboard/services/rgw_client.py

```diff
@@ -1083,7 +1083,15 @@ def get_bucket_replication(self, bucket_name, request=None):
                 if content.get('Code') == 'ReplicationConfigurationNotFoundError':
                     return None
             raise e
-
+    
+    @RestClient.api_post('?Action=CreateTopic&Name={topic_name}')
+    def create_topic(self,request=None,topic_name:str =None,push_endpoint: Optional[str]=None,opaqueData: Optional[str]=None, persistent: Optional[str]=None,time_to_live: Optional[str]=None,max_retries: Optional[str]=None,retry_sleep_duration: Optional[str]=None,policy: Optional[str]=None):
+        try:
+            result = request()  # type: ignore
```

Comment:
Do you think we need to generate a XML format for the request and provide it below?
```suggestion
            result = request(data=data)  # type: ignore
```